manifests/init.pp

```diff
@@ -174,6 +174,8 @@
 #
 # $dhcp_interface::             DHCP listen interface
 #
+# $dhcp_additional_interfaces:: Additional DHCP listen interfaces (in addition to dhcp_interface)
```

Comment:
Could you add that we don't provision any subnet for this by default (in contrast to ```dhcp_interface```)?

Reply:
Ok, will do.